doc/DEVELOPMENT.md

```diff
@@ -150,6 +150,9 @@ Using S3ClientConfig you can set up the following parameters for the underlying
   (max number of parts per upload is 10,000, minimum upload part size is 5 MiB).
   Part size must have **values between 5MiB and 5GiB.** Is set by default to **8MiB** (may change in future).
 
+* `unsigned(bool)`: Allows the usage of unsigned clients when accessing public datasets or when other mechanisms are
```

Comment:
nit:
```suggestion
* `unsigned(bool)`: Set to true to disable signing S3 requests.
```
(we don't want to suggest "other mechanisms" unless you're really, really sure)